tools/styles/tokens-v2/system-theme-bridge.css

```diff
@@ -631,13 +631,17 @@
     --system-tag-size-large-clear-button-spacing-inline-end: var(--spectrum-component-edge-to-visual-200);
     --system-textfield-background-color: var(--spectrum-gray-25);
     --system-textfield-background-color-disabled: var(--spectrum-gray-25);
-    --system-textfield-border-color: var(--spectrum-gray-500);
-    --system-textfield-border-color-hover: var(--spectrum-gray-600);
+    --system-textfield-border-color-hover: var(--spectrum-gray-400);
     --system-textfield-border-color-focus: var(--spectrum-gray-800);
     --system-textfield-border-color-focus-hover: var(--spectrum-gray-900);
     --system-textfield-border-color-keyboard-focus: var(--spectrum-gray-800);
-    --system-textfield-border-color-disabled: var(--spectrum-gray-300);
+    --system-textfield-border-color-disabled: var(--spectrum-disabled-border-color);
     --system-textfield-border-width: var(--spectrum-border-width-100);
+    --system-textfield-border-color: var(--spectrum-gray-300);
+    --system-textfield-spacing-block-start: var(--spectrum-component-padding-vertical-75);
+    --system-textfield-spacing-block-start-small: var(--spectrum-component-padding-vertical-100);
+    --system-textfield-spacing-block-start-large: var(--spectrum-component-padding-vertical-200);
+    --system-textfield-spacing-block-start-extra-large: var(--spectrum-component-padding-vertical-300);
```

Comment:
These paddings are defaulting to 0 because we don't yet have these tokens in SWC. They're meant to be used with the new Adobe Clean Spectrum VF (Variable Font), which we don't yet have available.

May I suggest instead the older tokens that were used before this? Our [Spectrum-CSS S2 implementation](https://spectrumcss.z13.web.core.windows.net/pr-2352/index.html?path=/story/components-text-field--default) uses the tokens that were in the previous spec, the `component-top-to-text` tokens that you're using for Spectrum 1/Express themes here. I think it'd be `component-top-to-text-75` for small, `-100` for medium, `-200` for large, and `-300` for XL.